classes/class3/README.md

```diff
@@ -1 +1,364 @@
+* `GET` returns a resource (such as an image or an html page). This is used for when your browser wants to read information from a server. Parameters and options are communicated in the url through the path (`/olin`) and query string parameters (`?pageid=12&button=2&coolsetting=super`) which you may recognize from various websites.
+* `POST` is used for when your browser wants to send information over to the server. For example, when you fill out an online form, that data is sent over to the server as `POST` data. This information is sent in the request header, is not visible in plain-text and can be encrypted.
+
+### Organizing an Express Application
+Express offers a tool called the `express-generator` to get a new directory up a running.
+```sh
+express expressintro
+
+   create : expressintro
+   create : expressintro/package.json
+   create : expressintro/app.js
+   create : expressintro/public
+   create : expressintro/public/images
+   create : expressintro/public/stylesheets
+   create : expressintro/public/stylesheets/style.css
+   create : expressintro/routes
+   create : expressintro/routes/index.js
+   create : expressintro/routes/users.js
+   create : expressintro/public/javascripts
+   create : expressintro/views
+   create : expressintro/views/index.jade
+   create : expressintro/views/layout.jade
+   create : expressintro/views/error.jade
+   create : expressintro/bin
+   create : expressintro/bin/www
+
+   install dependencies:
+     $ cd expressintro && npm install
+```
+This creates a series of directories `public`, `routes`, and `views`. The public directory is used for client side assets such as images, stylesheets, and client side javacript. Views are used to store template files which we will cover later. Routes, which we will focus on first, contain all the routing logic of our applications.
+More often then not, I find the generated app provides more structure and example files then you actually want and you end up deleting much of it. Feel free to explore the generator if you're curious, but for now we'll steal the folder structure and build the contents ourselves.
+
+Create this folder structure in the same directory as your `app.js` file by running `mkdir views routes public public/images public/javascripts public/stylesheets`. The create a new files `index.js` in the `routes` folder. This is where we will add some of the main routing logic for our app. You will often see another file like `users.js` in the routes folder which would contain routes that might manage login, logout, view profile, or other actions like that. You may even see `routes/cat.js` in your near future!
```

Comment:
The create a new files -> Then create a new file

`routes/cat.js` -> `/routes/cats.js` because why not